BlogTemplate/Pages/Edit.cshtml

```diff
@@ -38,7 +38,7 @@
                 <input type="submit" name="publish" id="postsave" class="btn btn-primary btn-sm" value="Publish" asp-page-handler="Publish" />
             </div>
             <div class="col-md-12">
-                <p class="cancel-link"><a href="/Index"><span class="glyphicon glyphicon-triangle-left"></span> Cancel and Undo Changes</a></p>
+                <p class="cancel-link"><a href ="@Model.PathToPost"><span class="glyphicon glyphicon-triangle-left"></span> Cancel and Undo Changes</a></p>

```

Comment:
1. the space between the attribute name (href) and the = is invalid.
2. Should this be using a tag helper to generate the link?

Reply:
@lucasisaza Look at the changes in #153 in Drafts.cshtml for a good example of using tag helpers to generate the link!